spec/models/comment_spec.rb

```diff
@@ -339,8 +339,13 @@
     end
 
     it "busts the comment cache" do
-      expect_any_instance_of(Comments::BustCacheWorker).to receive(:perform).with(comment.id)
-      comment.destroy
+      # here the comment is destroyed from the test above so we re-create one
+      new_comment = create(:comment, commentable: article)
+
+      # this replaces the use of expect_any_instance_of which is a RuboCop violation
+      sidekiq_assert_enqueued_with(job: Comments::BustCacheWorker, args: [new_comment.id]) do
+        new_comment.destroy
+      end
```

Comment:
I had to mess with this because RuboCop was complaining about the [previous use of `expect_any_instance_of`](https://github.com/thepracticaldev/dev.to/blob/b146528aa8f98e8f3a3c94bd56e25caa7c666d23/spec/models/comment_spec.rb#L342).

I also tried combining/chaining this test with the one above it to prevent the need for creating another comment. But when I did that, it was much harder to read and felt wrong since it was a single test for 2 different callbacks.

Maybe there's another way to test both callbacks on a single `comment.destroy` action?